src/Symfony/Bundle/FrameworkBundle/Console/Descriptor/JsonDescriptor.php

```diff
@@ -145,10 +146,20 @@ protected function describeContainerAlias(Alias $alias, array $options = array()
             return $this->writeData($this->getContainerAliasData($alias), $options);
         }
 
-        $this->writeData(
-            array($this->getContainerAliasData($alias), $this->getContainerDefinitionData($builder->getDefinition((string) $alias), isset($options['omit_tags']) && $options['omit_tags'], isset($options['show_arguments']) && $options['show_arguments'])),
-            array_merge($options, array('id' => (string) $alias))
-        );
+        try {
+            $this->writeData(
+                array($this->getContainerAliasData($alias), $this->getContainerDefinitionData($builder->getDefinition((string) $alias), isset($options['omit_tags']) && $options['omit_tags'], isset($options['show_arguments']) && $options['show_arguments'])),
+                array_merge($options, array('id' => (string) $alias))
+            );
+        } catch (ServiceNotFoundException $e) {
+            $targetId = (string) $alias;
+
+            if ('' !== $targetId && '@' === $targetId[0]) {
+                throw new ServiceNotFoundException($e->getId(), $e->getSourceId(), $e, array(substr($targetId, 1)));
```

Comment:
I'd suggest to not pass the previous exception as it provides basically nothing and it makes the console output harder to read (since both messages are displayed - same below):

![image](https://user-images.githubusercontent.com/243674/45820918-3b696e80-bce8-11e8-8242-881dcd6b2720.png)

Reply:
I have moved the exception handling to the command. It feels better to me to enhance the exception at the place where it is actually necessary and not somewhere in between where the code does not care about it. It looks like this now:

<img width="1297" alt="bildschirmfoto 2018-09-23 um 09 27 48" src="https://user-images.githubusercontent.com/1957048/45925312-fe34f480-bf12-11e8-9f09-f35ebd58d9ff.png">